submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/repositories/Repositories.kt

```diff
@@ -17,6 +17,9 @@ import org.springframework.data.mongodb.repository.MongoRepository
 import org.springframework.data.mongodb.repository.Query
 
 interface SubmissionMongoRepository : MongoRepository<DocSubmission, ObjectId> {
+    @Query(value = "{ 'accNo': '?0', 'version': { \$gte: 0 } }", fields = "{ relPath : 0 }")
+    fun findRelPath(accNo: String): String?
```

Comment:
Should we have an index to look by relPath or is this not an expensive db operation?

Reply:
Yes it will be needed but we are searching by accNo and version but retrieving only the relpath, no searching by relPath